main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"unicode"
+)
+
+func find(word string, words []string) int {
+	for i, w := range words {
+		if w == word {
+			return i
+		}
+	}
+	return -1
+}
+
+func IsSep(r rune) bool { return unicode.IsSpace(r) || unicode.IsPunct(r) || unicode.IsSymbol(r) }
+
+func words(s string) []string {
+	var w []string
+	var tmp []rune
+	wasSep := true
+	for _, r := range s {
+		if IsSep(r) {
+			if !wasSep {
+				w = append(w, string(tmp))
+				// tmp = tmp[:0]
+				tmp = nil
+			}
+			wasSep = true
+		} else {
+			wasSep = false
+			tmp = append(tmp, r)
+		}
+	}
+	if len(tmp) > 0 {
+		w = append(w, string(tmp))
+	}
+	return w
+}
+
+func wordOccurrences(s string) (string_of_words []string, occurrences []uint) {
```

Comment:
Please don't use variables with underscores in Go. In Go, variable names are camel case (https://en.wikipedia.org/wiki/Camel_case).